finagle-redis/src/main/scala/com/twitter/finagle/redis/Client.scala

```diff
@@ -38,79 +34,66 @@ class Client(service: Service[Command, Reply])
   with Sets
   with BtreeSortedSetCommands
   with HyperLogLogs
+  with PubSubs
+  with ServerCommands
+  with ConnectionCommands {
+  self: BaseClient =>
+}
 
-/**
- * Connects to a single Redis host
- * @param service: Finagle service object built with the Redis codec
- */
-class BaseClient(service: Service[Command, Reply]) {
+trait Transactions { self: Client =>
 
-  /**
-   * Authorizes to db
-   * @param password
-   */
-  def auth(password: ChannelBuffer): Future[Unit] =
-    doRequest(Auth(password)) {
-      case StatusReply(message) => Future.Unit
-    }
+  private[this] def singletonFactory: ServiceFactory[Command, Reply] =
```

Comment:
let's add parens for this function so it's clear that it's a function, and not an accessor.